InterviewFlashCards/QuestionsViewController.swift

```diff
@@ -54,23 +47,21 @@ class QuestionsViewController: UIViewController {
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any!) {
-        if (segue.destination is IFCAnswerViewController) {
-            (segue.destination as! IFCAnswerViewController).flashCard = flashCards[currentIndex]
+        if segue.identifier == segueIdentifier {
+            (segue.destination as! AnswersViewController).flashCard = flashCards[currentIndex]
```

Comment:
yeah, so i just wanted to track the identifier like you mentioned first, and then figure out the enum&switch statement in my next PR, if that's alright with you? just want to get this huge PR finally merged and will definitely implement the switch in my next one